docs/resources/credential.md

```diff
@@ -50,7 +50,26 @@ resource "databricks_credential" "external_mi" {
 }
 
 resource "databricks_grants" "external_creds" {
-  credential = databricks_credential.external.id
+  credential = databricks_credential.external_mi.id
+  grant {
+    principal  = "Data Engineers"
+    privileges = ["ACCESS"]
+  }
+}
+```
+
+For GCP
+
+```hcl
+resource "databricks_credential" "external_gcp_sa" {
+  name = "gcp_sa_credential"
+  databricks_gcp_service_account {}
+  purpose = "SERVICE"
```

Comment:
I thought you said it can only be used with purpose = "STORAGE"?

Reply:
addressed, thank you for catching it up